packages/auth/src/config/app.ts

```diff
@@ -29,5 +29,8 @@ export const Config = {
         ),
   keyRegistries: envStringArray('KEY_REGISTRIES', [
     'https://openpayments.network'
-  ])
+  ]),
+  interactDomain: envString('INTERACT_URL', 'http://localhost:3004'),
+  domain: envString('DOMAIN', 'http://localhost:3006'), // TODO: replace this with whatever frontend port ends up being
```

Comment:
Just a general question: What are these two?

Reply:
The `interactDomain` is the domain at which interaction takes place - essentially the domain of the RS. It's dawning on me that something like `resourceServerDomain` would be a better name.

`domain` is the domain of the AS. it's used to create things like the `continue` url when the AS needs to respond with a URL pointing to itself. Also realizing now that `authServerDomain` would be better.